src/AspNetCoreModuleV2/InProcessRequestHandler/inprocessapplication.cpp

```diff
@@ -201,169 +90,320 @@ IN_PROCESS_APPLICATION::SetCallbackHandles(
     _In_ VOID* pvShutdownHandlerContext
 )
 {
+    LOG_INFO(L"In-process callbacks set");
+
     m_RequestHandler = request_handler;
     m_RequestHandlerContext = pvRequstHandlerContext;
     m_ShutdownHandler = shutdown_handler;
     m_ShutdownHandlerContext = pvShutdownHandlerContext;
     m_AsyncCompletionHandler = async_completion_handler;
 
+    m_blockManagedCallbacks = false;
+    m_Initialized = true;
```

Comment:
You can remove one of these bools I believe.

Reply:
Not really they control different things.

Reply:
m_Initialized isn't initialized.